src/apollo/client.ts

```diff
@@ -72,5 +85,7 @@ export const blockClient: { [chainId in ChainId]: ApolloClient<NormalizedCacheOb
   [ChainId.GÖRLI]: ropstenBlockClient,
   [ChainId.KOVAN]: ropstenBlockClient,
   [ChainId.MATIC]: maticBlockClient,
-  [ChainId.MUMBAI]: mumbaiBlockClient
+  [ChainId.MUMBAI]: mumbaiBlockClient,
+  [ChainId.BSCTESTNET]: mumbaiBlockClient,
```

Comment:
This will cause errors for both dmm-interface and dmm-info.

We need block clients for both bsctestnet and bscmainnet